WordPress/Classes/ViewRelated/Reader/ReaderPostMenu.swift

```diff
@@ -48,19 +50,27 @@ open class ReaderPostMenu {
                                                 if let topic: ReaderSiteTopic = self.existingObject(for: topic.objectID, context: topic.managedObjectContext) {
                                                     self.toggleSubscribingNotifications(for: topic)
                                                 }
-            })
+                                               })
         }
 
         // Following
-        let buttonTitle = post.isFollowing ? ReaderPostMenuButtonTitles.unfollow : ReaderPostMenuButtonTitles.follow
-        alertController.addActionWithTitle(buttonTitle,
+        alertController.addActionWithTitle(post.isFollowing ? ReaderPostMenuButtonTitles.unfollow : ReaderPostMenuButtonTitles.follow,
             style: .default,
             handler: { (action: UIAlertAction) in
                 if let post: ReaderPost = self.existingObject(for: post.objectID, context: post.managedObjectContext) {
                     self.toggleFollowingForPost(post, viewController)
                 }
         })
 
+        // Seen
+        alertController.addActionWithTitle(post.isSeen ? ReaderPostMenuButtonTitles.markUnseen : ReaderPostMenuButtonTitles.markSeen,
```

Comment:
Should we put this behind the unread counts feature flag so we can flip a switch and turn both features on at once?

Reply:
We could. I went back and forth, but ultimately didn't because I figured the ability to toggle seen could be useful standalone and released without the count. If you prefer otherwise, I can do it in my next PR (which I'm working on now to add this functionality to reader cards.)

Reply:
Nevermind. I'll go ahead & do it.